Documentation/metrics.md

```diff
@@ -32,6 +33,8 @@ is totally unavailable.
 
 `leader_changes_seen_total` counts the number of leader changes the member has seen since its start. Rapid leadership changes impact the performance of etcd significantly. It also signals that the leader is unstable, perhaps due to network connectivity issues or excessive load hitting the etcd cluster.
 
+`proposals_committed_total` counts the total number of consensus proposals committed. This counter should be seen to be increasing over time if a cluster is healthy. Healthy members of an etcd cluster can still have different number of committed proposals at the same time. A member may be recovering from peers after startup, or lagging behind the leader in the cluster, or it may be up-to-date and participating fully, and it may be the leader. It is important to monitor this metric across all the members in the cluster. A consistently large lag between a member and its leader indicates the member is slow or unhealthy.
```

Comment:
This counter should increase over time if the cluster is healthy. Several healthy members of an etcd cluster may have a different total committed proposals at once. This discrepancy may be due to recovering from peers after starting, lagging behind the leader, or being the leader and therefore having the most commits. It is important to monitor this metric across all the members in the cluster; a consistently large lag between a single member and its leader indicates that member is slow or unhealthy.